bedrock/externalpages/templates/externalpages/pocket/includes/mobile-nav.html

```diff
@@ -0,0 +1,45 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="mobile-nav-wrapper" tabindex="-1">
+    <nav class="mobile-nav" role="dialog" aria-modal="true" aria-label="Pocket Mobile Menu">
+        <div class="mobile-nav-close">
+            <button class="mobile-nav-close-btn" data-tooltip="Close">Close the Pocket Mobile Menu</button>
+        </div>
+        <ul class="mobile-nav-list">
+            <li class="mobile-nav-list-item">
+                <a class="mobile-nav-list-link home" href="https://getpocket.com/home?src=navbar" id="global-nav-home-link">Home<span class="beta">BETA</span></a>
+            </li>
+            <li class="mobile-nav-list-item">
+                <a class="mobile-nav-list-link my-list" href="https://getpocket.com/my-list?src=navbar" id="global-nav-collections-link">My List</a>
```

Comment:
This is a duplicate ID that matches with the `Collections` list item. Let's update it to be unique and reflect the `My List` item. 

```suggestion
                <a class="mobile-nav-list-link my-list" href="https://getpocket.com/my-list?src=navbar" id="global-nav-user-list-link">My List</a>
```